Settings.php

```diff
@@ -185,4 +193,29 @@ private function createTrackToCustomSiteIdSetting()
         $this->addSetting($this->customPiwikSiteId);
     }
 
+    private function anonymizeCustomPiwikCheckbox()
+    {
+        $this->anonymizeCustomPiwik = $this->anonymizationOption("anonymizeCustomPiwik");
+
+        $this->addSetting($this->anonymizeCustomPiwik);
+    }
+
+    private function anonymizeSelfPiwikCheckbox()
+    {
+        $this->anonymizeSelfPiwik = $this->anonymizationOption("anonymizeSelfPiwik");
+
+        $this->addSetting($this->anonymizeSelfPiwik);
+    }
+
+    private function anonymizationOption($optionName)
+    {
+        $optionCheckbox = new SystemSetting($optionName, 'Turn on anonymization');
```

Comment:
Maybe it would be better to say `Anonymize tracking requests` or only `Anonymize` or `Use anonymization`